modules/control-panel/Module.php

```diff
@@ -9,6 +10,8 @@ class Module extends \yii\base\Module
 {
     public function init()
     {
+        // Define a custom alias named after the namespace
+        Craft::setAlias('@biglotteryfund', __DIR__);
```

Comment:
Came across this somewhere in Craft's github issues for newly-upgraded installs – was causing `./craft` commands to fail without it.

Reply:
Interesting, I spotted that error too. Is this module the right place for this line?

Reply:
(We don't need to answer this for this PR, merge away)